packages/twenty-front/src/modules/navigation/components/MobileNavigationBar.tsx

```diff
@@ -67,5 +68,10 @@ export const MobileNavigationBar = () => {
     },
   ];
 
+  // Check if the user is logged in
```

Comment:
we do not leave comments in the code, it's usually not useful to understand the behaviors, the code should be self explanatory.